src/Funnel.sol

```diff
@@ -35,72 +46,79 @@ contract Funnel is IFunnel, NativeMetaTransaction, MetaTxContext, Initializable
     // owner => spender => renewableAllowance
     mapping(address => mapping(address => RenewableAllowance)) rAllowance;
 
-    /*//////////////////////////////////////////////////////////////
-                            EIP-2612 STORAGE
-    //////////////////////////////////////////////////////////////*/
+    //////////////////////////////////////////////////////////////
+    ///                        EIP-2612 STORAGE
+    //////////////////////////////////////////////////////////////
 
-    uint256 internal INITIAL_CHAIN_ID;
+    // initial_chain_id to be set during initiailisation
+    uint256 internal initial_chain_id;
 
-    bytes32 internal INITIAL_DOMAIN_SEPARATOR;
+    // initial_domain_separator to be set during initiailisation
+    bytes32 internal initial_domain_separator;
 
+    // constant for the given struct type that do not need to be runtime computed. Required for EIP712-typed data
     bytes32 internal constant PERMIT_RENEWABLE_TYPEHASH =
         keccak256(
             "PermitRenewable(address owner,address spender,uint256 value,uint256 recoveryRate,uint256 nonce,uint256 deadline)"
         );
 
+    // constant for the given struct type that do not need to be runtime computed. Required for EIP712-typed data
```

Comment:
redundant comment, would only enforce comment for external & public functions/state

Reply:
This was flagged out during audit, just following it and patching the holes and gaps.